framework/src/types.ts

```diff
@@ -160,11 +160,25 @@ interface RPCConfig {
 	port: number;
 }
 
+export interface Generator {
+	readonly address: string;
+	readonly encryptedPassphrase: string;
+}
+
+export interface GeneratorConfig {
+	waitThreshold: number;
+	generators: Generator[];
+	force?: boolean;
+	defaultPassword?: string;
+	modules: Record<string, unknown>;
+}
```

Comment:
These two look very odd to each other. If `Generator` is an account then `GeneratorConfig` relates to account config, which is not. We can change `GeneratorConfig` to `GenerationConfig` or `Generator` to `GeneratorAccount`. Not sure either.

Reply:
`GenerationConfig` sounds good! will update